### PR TITLE
Implement filter goto action support

### PR DESCRIPTION
**Pull request by e0ne**

### CODE REVIEW

**Comment by aboch** [comment]
Is it possible to add UT for this newly added option?

File: filter_test.go [hunk 1]
```diff
@@ -1726,6 +1732,15 @@ func TestFilterFlowerAddDel(t *testing.T) {
 		t.Fatal("Mirred action isn't TC_ACT_STOLEN")
 	}
 
+	ga, ok := flower.Actions[1].(*GenericAction)
+	if !ok {
+		t.Fatal("Unable to find mirred action")
```
Comment (adrianchiris) [inline filter_test.go:1737]: generic action ?

**Comment by e0ne** [comment]
> Is it possible to add UT for this newly added option?
> 
> 

Unit/tests added

**Comment by e0ne** [comment]
@aboch @vishvananda  could you please take a look on this PR once you have a time?

**Comment by e0ne** [comment]
just a kindly ask to @aboch @vishvananda for review

**Comment by aboch** [comment]
LGTM

**Pull request merged**